src/builder.rs

```diff
@@ -9,6 +9,8 @@ pub struct Builder {
 
     config: Config,
 
+    network: IpNetwork,
```

Comment:
How about the following?

```suggestion
    ip_version: IpVersion
```

Reply:
i think this one is good